x-parser.js

```diff
@@ -208,10 +118,8 @@ export class XParser {
   // Examples:
   //  - ok: <h6>, <my-element-1>
   //  - not ok: <-div>, <1-my-element>
-  // TODO: #236: Restrict tags to lowercase once <svg> support is removed.
-  // Svg tag names follow the above rules, but may have capital letters.
-  #openTagStart = /<(?![0-9A-Z-])[a-zA-Z0-9-]+(?<!-)(?=[\s\n>])/y;
-  #closeTag =   /<\/(?![0-9A-Z-])[a-zA-Z0-9-]+(?<!-)>/y;
+  #openTagStart = /<(?![0-9-])[a-z0-9-]+(?<!-)(?=[\s\n>])/y;
```

Comment:
This is the reason we get into a different error range. We simply don’t look for _any_ upper case letters in tag names / attribute names anymore.